gf-webapp-admin/src/component/sideMenu/addRoomModal/adults/Adults.tsx

```diff
@@ -1,29 +1,29 @@
-import {useContext, useEffect, useState} from "react";
-import {AddRoomContext} from "src/component/sideMenu/addRoomModal/addRoomContext";
+import {SmallTitle} from "gf-ui-lib/src/components/SmallTitle/SmallTitle";
+import {useState} from "react";
+import {useAddRoomContext} from "src/component/sideMenu/addRoomModal/addRoomContext";
 
 
 export const Adults = () => {
-  const {roomStartState} = useContext(AddRoomContext);
-  const [adults, setAdults] = useState(roomStartState.adults);
+  const {room, setRoom} = useAddRoomContext();
+  const [adults, setAdults] = useState(room.adults);
```

Comment:
Why do you need to put adults in state if it is already in context?

Reply:
i'll implement this in #43 issue